### DISCUSSION TITLE
Help us rebrand! We're seeking a new name for z/OS Open Tools that we can trademark and call our own.

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#767] [General]
Help us choose a new name for z/OS Open Tools by participating in this poll: https://app.sli.do/event/aT3aeJRVSdp9BQ49AmYxaf/embed/polls/5999e401-2329-4607-89e5-dc645a51967d

Poll will close on **Friday, May 31**

If you're wondering why we're changing our name, "z/OS" and "Z" are IBM trademarks, and the Open Mainframe Project is not affiliated with IBM.

**Comment by catherdersoftware** [comment]
Is there a way to see the current vote numbers after voting? I've already voted, but I'm curious as to where the general feelings are.

**Comment by IgorTodorovskiIBM** [comment]
Results so far:
<img width="1895" alt="Screenshot 2024-05-28 at 12 32 15 PM" src="https://github.com/ZOSOpenTools/meta/assets/39890068/3041f62e-4dfe-4842-b5ac-f859f67c5530">

**Reply by MikeFultonDev** [reply]
Note there appears to be a 'ZOT' already: https://github.com/project-zot/zot

**Reply by glen-21cs** [reply]
Only a couple letters away from MOTH or MOTHER - any backroynm geniuses out there. ;)

**Comment by ijmitch** [comment]
I'm afraid I missed the Discord discussions.

I might be sensitized from working with the wrong sort of people (am I allowed to say that?) but there are folks who take 'mainframe' to be all sorts of things - IBM Z, Fujitsu, even IBM i - so I'm having a hard time thinking to vote for one of those options. Maybe that's neutralized by being part of OMP now though.

The option with 'Ported' in the name seems to have support too - personally I don't think including that word adds much and I wonder if it has vaguely negative connotations about "ported (... so not really 'native')" which would certainly be an unjust inference.
